.gradient/workflows/training-pipeline.yaml

```
@@ -0,0 +1,225 @@
+            type: dataset
+            with:
+              ref: training-environment-setup
+            
+      - name: data-preparation
+        uses: script@v1
+        needs: [setup-environment]
+        with:
+          script: |
+            # Data preparation step
+            echo "Starting data preparation..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)